modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_data_set_views.scss

```diff
@@ -0,0 +1,21 @@
+.data-views-list-sheet {
+
+    background-color: $white;
+    border-color: $border-color;
+    border-style: solid;
+    border-width: $border-width;
+    border-radius: map-get($spacers, 1);
+
+    & .data-views-list-header {
+        padding-top: map-get($spacers, 4);
+        padding-right: map-get($spacers, 4);
+        padding-left: map-get($spacers, 4);
+        border-bottom: 1px solid $gray-200;
+    }
+
+    & .list-group {
+        padding-top: map-get($spacers, 4);
+        padding-right: map-get($spacers, 4);
+        padding-left: map-get($spacers, 4);
+    }
+}
```

Comment:
Hmm, needing this custom CSS is a bit of a smell. Can you utilize [Clay components](https://clayui.com/docs/components/) instead of having to write custom markup like this?